src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/MinerAndroid.java

```diff
@@ -22,21 +25,33 @@
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 import me.mrCookieSlime.Slimefun.api.inventory.BlockMenu;
 
+/**
+ * Represents a {@link MinerAndroid Programmable Android (Miner)}.
+ * This {@link ProgrammableAndroid} can dig blocks and move.
+ *
+ * @author Linox
```

Comment:
You are not the sole author of this class...

Reply:
> You are not the sole author of this class...

Well idk who else is so I just added mine.

Reply:
`git blame`, also me most likely.

Reply:
Android has no git